src/main/java/frc/robot/subsystems/elevator/ElevatorIOReal.java

```diff
@@ -0,0 +1,89 @@
+package frc.robot.subsystems.elevator;
+
+import com.ctre.phoenix6.StatusSignal;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.ForwardLimitValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.ctre.phoenix6.signals.ReverseLimitValue;
+import dev.doglog.DogLog;
+
+public class ElevatorIOReal implements ElevatorIO {
+
+  private boolean EmergencyMode;
+  private TalonFX m_FrontMotor =
+      new TalonFX(ElevatorConstants.FrontElevatorMotorID, ElevatorConstants.ElevatorMotorCanBus);
+  private TalonFX m_BackMotor =
+      new TalonFX(ElevatorConstants.BackElevatorMotorID, ElevatorConstants.ElevatorMotorCanBus);
+  private double FrontMotorPos = m_FrontMotor.getRotorPosition().getValueAsDouble();
```

Comment:
These don't get updated anywhere yet.  Should that be in the `update` function below?